src/components/ListView.jsx

```diff
@@ -24,18 +25,25 @@ function ListView() {
       //Used to remove the snapshot listener when the component is unmounted/unsubscribed
       unsubscribe();
     };
-  }, []);
+  }, [localToken, navigate]);
 
   return (
-    <div>
-      Your Shopping List:
-      <ul>
-        {items.map((item, idx) => (
-          <li
-            key={idx}
-          >{` Name: ${item.itemName}  Frequency: ${item.frequency}`}</li>
-        ))}
-      </ul>
+    <div className="m-20">
+      {!items.length ? (
+        <Welcome items={items} />
```

Comment:
Is it any reason why Welcome component getting props? Any plans to use it later?